src/examples/text-inputs/Size.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Hint, Input } from '@zendeskgarden/react-forms';
+
+const Example = () => (
+  <Grid>
+    <Row>
+      <Col>
+        <Field>
+          <Label>Plant</Label>
+          <Hint>Plants in the Garden</Hint>
```

Comment:
Recommend dropping the `Hint` here in order to make the example easier to compare.

Reply:
This suggestion conflicts with the Google doc. Should we confirm with @ginnywood first?

Reply:
This was only a mistake we had a comment to correct in the final image assets.

Reply:
Removed 👍